src/libstd/sys_common/mod.rs

```diff
@@ -34,7 +34,6 @@ pub mod condvar;
 pub mod io;
 pub mod memchr;
 pub mod mutex;
-pub mod net;
```

Comment:
Because Redox does networking using files, and does not support the traditional Berkeley Socket API, we must specify an alternative net module, which is defined in `sys::redox::net` and imported down below